RxSwift/Disposables/CompositeDisposable.swift

```diff
@@ -47,6 +49,26 @@ public class CompositeDisposable : DisposeBase, Disposable, Cancelable {
     /**
      Initializes a new instance of composite disposable with the specified number of disposables.
     */
+    public init(_ disposable1: Disposable, _ disposable2: Disposable, _ disposable3: Disposable, _ disposable4: Disposable) {
```

Comment:
Hi @larryonoff ,

I've meant to merge this one `public init(_ disposable1: Disposable, _ disposable2: Disposable, _ disposable3: Disposable, _ disposable4: Disposable)` with `public init(_ disposables: Disposable...) {`

into `public init(_ disposable1: Disposable, _ disposable2: Disposable, _ disposable3: Disposable, _ disposable4: Disposable, _ disposables: Disposable...)`

If you could squash those these commits and make a PR to `develop` branch, I would be happy to merge this.